libs/ledger-live-common/src/families/stellar/js-synchronization.ts

```diff
@@ -107,10 +107,19 @@ const getAccountShape: GetAccountShape = async (info, syncConfig) => {
   };
   return {
     ...shape,
-    operations: allOperations.map((op) => ({
-      ...op,
-      subOperations: inferSubOperations(op.hash, subAccounts),
-    })),
+    operations: allOperations.map((op) => {
+      const subOperations = inferSubOperations(op.hash, subAccounts);
+      const isSubAccount = subOperations.length > 0;
+
+      return {
+        ...op,
+        // Type FEES for token account operations (shows on main account only)
+        type: isSubAccount ? "FEES" : op.type,
+        // Fees should not have value
+        value: isSubAccount ? new BigNumber(0) : op.value,
+        subOperations,
+      };
+    }),
```

Comment:
I'm not sure if this is the best place where to set main account sub-account operations as FEES, but it works (we don't want to change the sub-account operation type).

I tried to update operation `id` here with `-FEES`, but it converted sub-account operations as fees as well, which is not what we want. It looks like there might be a conversion somewhere down the line. :thinking: